packages/core/botpress/src/web/views/FlowBuilder/sidePanel/flows/index.jsx

```diff
@@ -47,6 +41,21 @@ const getInitialState = currentFlow => {
 export default class FlowsList extends Component {
   menuButtons = {}
 
+  componentDidUpdate(props) {
+    const { currentFlow } = props
```

Comment:
Just asking: no need for this to be also called in componentDidMount? For ref, componentDidUpdate is not called on initial render

Reply:
OK cool, though I usually prefer to stay on the safe side and call the same method from both componentDidMount and componentDidUpdate. You have the proper check for currentFlow so if it's not defined by the time component is mount (I think it's undetermined if props are defined or not by that time) it will do nothing

Reply:
okey will fix tommorow)) ty